spec/controllers/high_voltage/pages_controller_spec.rb

```diff
@@ -3,16 +3,13 @@
 RSpec.describe HighVoltage::PagesController do
   render_views
 
-  # Test that each static page is generated correctly
-  Dir.glob(Rails.root.join('app', 'views', 'pages', '*.erb')).each do |file|
-    page_name = file.split('/').last.split('.').first
-
+  %w[ cookies terms_and_conditions unsubscribe ].each do |page_name|
     describe "when rendering #{page_name}" do
       before do
         get :show, id: page_name
       end
 
-      it { should respond_with(:success) }
+      specify { expect(response.status).to eq(200) }
```

Comment:
> Everyone knows what 200 means, `:ok` is unnecessary indirection.

For 200, yes. But can everyone remember which is which out of `301` and `302`? Or the various `4xx` codes? I have to check every time.

I don't think this is indirection so much as description. It expresses the intention of the test better, and reports the result more clearly. There's no harm in it, and we'll get more helpful error messages.

Compare this:

```ruby
expect(response.status).to eq(502)
```

```
Failure/Error: expect(response.status).to eq(502)

  expected: 502
       got: 200

  (compared using ==)
```

with this:

```ruby
expect(response).to have_http_status(:bad_gateway)
```

```
Failure/Error: expect(response).to have_http_status(:bad_gateway)
  expected the response to have status code :bad_gateway (502) but it was :ok (200)
```

I much prefer the latter.